ethereum/jsonrpc/src/test/resources/tech/pegasys/pantheon/ethereum/jsonrpc/debug_accountRangeAt_complete.json

```diff
@@ -0,0 +1,27 @@
+{
+  "request": {
+    "id": 414,
+    "jsonrpc": "2.0",
+    "method": "debug_accountRangeAt",
```

Comment:
This method needs a rename for the tests too

Reply:
fixed, temporarily.  Retesteth is changing the name but is unstable where they changed it.